### ISSUE TITLE
Tests do not run on a fresh checkout of repository.

### ISSUE THREAD

**Issue by taylorhadden** [typewriter-editor/typewriter#69]
I recently checked out the repo for the purposes of #68. When I try to run the tests with `npm run test`, I get this message:

```
> TS_NODE_PROJECT=test/tsconfig.json mocha

Error: No test files found
```

**Comment by jacwright** [comment]
Does anyone else get this error? I'm wondering if @taylorhadden is the only one who can't run the test, or if I am the only one who *can*.

**Comment by taylorhadden** [comment]
I experimented with modifying the test script to `TS_NODE_PROJECT=test/tsconfig.json mocha -r ts-node/register 'test/**/*.test.*'` (based on [this](https://stackoverflow.com/questions/54805319/mocha-error-no-test-files-found-test-npm-err-test-failed).

Running `npm run test` at this point gets something going, but it errors out with `Event is not defined` in `decorations.ts`. This seems like the tsconfig's 'dom' lib definition is not being loaded correctly...

**Comment by HemalR** [comment]
I tried it out and got the same results. I.e.

A message `Error: No test files found` and modifying it as per @taylorhadden the same `Event is not defined` message.

**Comment by jacwright** [comment]
Fixed by 506d444
